### DISCUSSION TITLE
Manual IP range for Host-Only on MacOS M1

### DISCUSSION THREAD

**Discussion by rafnizw** [utmapp/UTM#4932] [UTM Help (macOS)]
Hi 

Does anyone know how to permanently set custom IP range for Host-Only network on MacOS Silicon M1? 
I can see the IP is set under /Library/Preferences/SystemConfiguration/com.apple.vmnet.plist
However any modification of this file is reverted automatically to previous IP

Thanks for

**Comment by zsoro2** [comment]
I have the same problem. What I usually do:

1, Start the UTM  VM (IP gets updated)
2, Stop the VM
3, I rewrite the IP addresses in two place
- /var/db/dhcpd_leases ( Shared_Net_Address to 192.168.56.1 ) 
- /Library/Preferences/SystemConfiguration/com.apple.vmnet.plist ( IP to 192.168.56.55 )
4, I restart my computer
5, I checked the two files, they should still contain good IP addresses. 
6, I start the UTM VM again. And now IP usually stays static, with no rewrite.

It's not a bulletproof solution but usually works for me. 

Let me know if anyone has a proper solution.
